_docs-sources/guides/stay-up-to-date/cis/cis-1.5.0/deployment-walkthrough/step-2-update-references-to-the-gruntwork-infrastructure-as-code-library.md

```diff
@@ -0,0 +1,256 @@
+---
+sidebar_label: Update references to the Gruntwork Infrastructure as Code Library
+---
+
+# Step 2: Update references to the Gruntwork Infrastructure as Code Library
+
+To update to the CIS AWS Foundations Benchmark v1.5.0, you need to update your references to the Gruntwork
+Infrastructure as Code Library to use compatible versions. We (Gruntwork) have reviewed and updated all the library modules for compatibility with the new version of the benchmark. As a customer, you need to update to the correct versions of the Gruntwork library to pick up the changes necessary for your infrastructure to be compatible.
+
+The following table provides a summary of all the relevant Gruntwork AWS modules and the respective versions that are
+compatible with CIS AWS v1.5.0:
+
+<a id="compatibility-table" class="snap-top"></a>
+
+<table id="compatibility-table">
+  <colgroup>
+    <col />
+    <col />
+    <col />
+  </colgroup>
+  <tbody>
+    <tr className="odd">
+      <td>
+        <p>Gruntwork Repo</p>
+      </td>
+      <td>
+        <p>
+          <strong>Minimum version with CIS AWS v1.5.0 support</strong>
+        </p>
+      </td>
+    </tr>
+    <tr className="even">
+      <td>
+        <p>terraform-aws-security</p>
+      </td>
+      <td>
+        <p>
+          <strong>
+            <a href="https://github.com/gruntwork-io/terraform-aws-security/releases/tag/v0.67.2">
+              v0.67.2
+            </a>
+          </strong>
+        </p>
+      </td>
+    </tr>
+    <tr className="odd">
+      <td>
+        <p>terraform-aws-cis-service-catalog</p>
+      </td>
+      <td>
+        <p>
+          <strong>
+            <a href="https://github.com/gruntwork-io/terraform-aws-cis-service-catalog/releases/tag/v0.42.9">
+              v0.42.9
+            </a>
+          </strong>
+        </p>
+      </td>
+    </tr>
+  </tbody>
+</table>
+
+## 2A Updating with Patcher
+
+Beginning with CIS AWS Foundations Benchmark v1.5.0, Gruntwork is providing an automated way update to the correct versions of the Gruntwork library using a tool we call Patcher.
+
```

Comment:
We need to make it a bit clearer it only works with the CIS Ref Arch (not just random CIS code). Perhaps add it to the compatibility table? E.g., Make the first column "Product" and the first row "Ref Arch"...